pkg/config/default.go

```diff
@@ -46,7 +46,7 @@ var (
 	// DefaultInitPath is the default path to the container-init binary
 	DefaultInitPath = "/usr/libexec/podman/catatonit"
 	// DefaultInfraImage to use for infra container
-	DefaultInfraImage = "k8s.gcr.io/pause:3.5"
+	DefaultInfraImage = "localhost/podman-pause:Version-Built"
```

Comment:
Please remove :Version-Built
Does this work when  you attempt to use it?

Reply:
Is only a string, as said, the simplest solution

Reply:
But if this default is  used somewhere it will fail.

Reply:
But it should not be.

pod_create.go :: func pullOrBuildInfraImage
```
...
	if imageName != config.DefaultInfraImage {
		_, err := rt.LibimageRuntime().Pull(context.Background(), imageName, config.PullPolicyMissing, nil)
		if err != nil {
			return err
		}
	} else {
		name, err := buildPauseImage(rt, rtConfig)
		if err != nil {
			return fmt.Errorf("building local pause image: %w", err)
		}
		imageName = name
	}
...
```

Reply:
I think the default should be ""
Then if the user does not enter an imageName, it will use buildPauseImage.

Reply:
The info of the default image would disappear, but also a little user experience and the output should than be

```
$ podman pod create --help | grep infra-image
      --infra-image string            The image of the infra container to associate with the pod
```

Reply:
Change the help to state 
Image to use for infra container, rather then builtin.

Reply:
This would be a change of containers/podman/.../pods/create.go in line 78. Can you do this? I hope.